tests/system/ak_test.go

```diff
@@ -75,6 +75,7 @@ func setUpTest(t *testing.T, akPath string) string {
 
 	// Redirect the OS's stdout and stderr through a pipe, to
 	// detect when the AK server is ready for the test to begin.
+	// TODO: Replace with /healthz or /readyz once they're dependable.
```

Comment:
ticket? also i think they're dependable right now for this use

Reply:
I prefer to use opentelemetry's endpoint, but you're right, we can use the one we currently have too. Will do in a separate PR.

Reply:
done (added ticket ID)